### PR TITLE
Add spacing between first and second parameter name

### LINKED ISSUE #1505: `BasicFormat` doesn't add spaces between first and second parameter name
### Description

When using `FunctionParameterSyntax` with `SwiftSyntaxBuilder`, the build result (formatted by `BasicFormat`) doesn't have spaces between first and second parameter names, which is "lexically legal" but far against user intention.

### Steps to Reproduce

```swift
let parameter = FunctionParameterSyntax(firstName: "on", secondName: "eventLoop", colon: .colonToken(), type: TypeSyntax("EventLoop"))
print(parameter) // oneventLoop: EventLoop
```

### PR DESCRIPTION
**Pull request by kimdv**
Fixes #1505

### CODE REVIEW

**Comment by kimdv** [comment]
@swift-ci please test

**Comment by kimdv** [comment]
@stevapple

File: Tests/SwiftSyntaxBuilderTest/FunctionParameterSyntaxTests.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+import SwiftSyntax
+import SwiftSyntaxBuilder
+
+final class FunctionParameterSyntaxTests: XCTestCase {
+  func testFunctionParameterSyntaxSpacing() {
+    let builder = FunctionParameterSyntax(firstName: "on", secondName: "eventLoop", colon: .colonToken(), type: TypeSyntax("EventLoop"))
```
Comment (stevapple) [inline Tests/SwiftSyntaxBuilderTest/FunctionParameterSyntaxTests.swift:19]: I think this should work now
```suggestion
    let builder = FunctionParameterSyntax(firstName: "on", secondName: "eventLoop", type: TypeSyntax("EventLoop"))
```

**Comment by kimdv** [comment]
@swift-ci please test

**Review by stevapple — APPROVED** [review]
Thanks!

**Comment by ahoppen** [comment]
@swift-ci Please test

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

**Pull request merged**